apps/dashboard/test/system/jobs_app_test.rb

```diff
@@ -274,4 +290,153 @@ def script_yml(dir)
       assert_nil YAML.safe_load(File.read("#{script_dir}/1_job_log"))
     end
   end
+
+  test 'editing scripts initializes correctly' do
+    Dir.mktmpdir do |dir|
+      setup_project(dir)
+      setup_script('1')
+
+      visit project_path('1')
+      find('[href="/projects/1/scripts/1/edit"]').click
+
+      click_on('Add new option')
+      new_field_id = 'add_new_field_select'
+
+      actual_new_options = page.all("##{new_field_id} option").map(&:value).to_set
```

Comment:
```suggestion
      actual_new_options = page.all("#{new_field_id} option").map(&:value).to_set
```

Reply:
I can try this, but IDK if it'll work. the first `#` is the ID CSS selector. The second is that string interpolation function for `#{new_field_id}`. So we want the resulting string (after interpolation) would be be `"#add_new_field_select option"`

Reply:
Yea the CSS query doesn't work without that first #.